### PR TITLE
test: add test for `goto_definition` for label

### PR DESCRIPTION
**Pull request by Eric-Song-Nop**
I hope to implement `goto_reference` with labels, here is where I get started

### CODE REVIEW

File: crates/tinymist-query/src/goto_definition.rs [hunk 1]
```diff
@@ -65,12 +66,28 @@ mod tests {
         snapshot_testing("goto_definition", &|world, path| {
             let source = world.source_by_path(&path).unwrap();
 
+            let docs = find_module_level_docs(&source).unwrap_or_default();
+            let properties = get_test_properties(&docs);
+            let must_compile = properties
```
Comment (Myriad-Dreamin) [inline crates/tinymist-query/src/goto_definition.rs:71]: could you lift the repeated code as a function in `tests.rs`? I guess you copied it from `completion.rs`.

**Review by Myriad-Dreamin — COMMENTED** [review]
Wonderful. You've mastering the magic notations (comments) in the snapshot files without any guidance. This was what I mostly want to documenting for developers of `tinymist-query`.

> I hope to implement `goto_reference` with labels, here is where I get started.

Sounds like a nice start. Please feel free to ask me via github, discord, or email if you have any question.

File: crates/tinymist-query/src/goto_definition.rs (hunk 1 above)
Comment (Eric-Song-Nop) [reply crates/tinymist-query/src/goto_definition.rs]: sure, I made a very light abstract there

**Comment by Eric-Song-Nop** [comment]
I made a very simple implementation for goto_reference for labels, and it just works, I can split this pr into two if you find that better. BTW, may I ask where can I find ur discord channel?

**Comment by Myriad-Dreamin** [comment]
> I can split this pr into two if you find that better.

Let us split it.

>  BTW, may I ask where can I find ur discord channel?

- [Typst Discord Server > universe > Tinymist](https://discord.com/channels/1054443721975922748/1260973804637786224)

  Go to there if we are going to discuss features for tinymist.

- [Typst Discord Server > forge > IDE](https://discord.com/channels/1054443721975922748/1229699205786898485)

  Go to there if we are going to continue pushing features to typst/typst. Note: some features are not needed, and they prefer minimal implementations.

**Comment by Myriad-Dreamin** [comment]
Thank you.

**Pull request merged**